Nasal/MCDU/VERTREV.nas

```diff
@@ -61,10 +61,16 @@ var vertRev = {
 			me.L3 = [" [    ]", " SPD CSTR", "blu"];
 			me.L4 = [" CONSTANT MACH", nil, "wht"];
 			me.L5 = [" WIND DATA", nil, "wht"];
-			me.L6 = [" RETURN", nil, "wht"];
+			me.L6 = [" CLB", nil, "amb"];
 			me.R2 = ["RTA ", nil, "wht"];
 			me.R3 = ["[      ] ", "ALT CSTR  ", "blu"];
-			me.arrowsMatrix = [[0, 0, 0, 1, 1, 1], [0, 1, 0, 0, 0, 0]];
+			me.R6 = ["DES ", nil, "amb"];
+			# When the system does vertical planning, L6 should be RETURN and R6 not used if the MCDU knows the waypoint is during climb or descent.
+			# The CLB or DES prompts should only be shown for a vertical revision in the cruise phase.
+			# For now we fake it and allow the user to press either, which both act like RETURN.
+			# When CLB/DES are shown, a small "OR" should be shown between them.
+			# The 'arrows' for CLB/DES should actually be asterisks.
```

Comment:
Asterisks require changes to the .svg . I will progressively add these to the missing pages. I know other pages need changes too.